packages/select/src/components/select/multiSelect.tsx

```diff
@@ -196,6 +202,13 @@ export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IM
     ) => {
         return (e: React.KeyboardEvent<HTMLElement>) => {
             const { which } = e;
+            const htmlTarget = e.target as HTMLElement;
+            if (htmlTarget.className.includes("remove")) {
```

Comment:
we shouldn't use "magic" string constants like this. instead we should use predictable variables